edit.php

```diff
@@ -31,12 +26,12 @@
 {
 	ImmersiveFormTemplate::AddImmersiveDialog('An error occurred', IMMERSIVE_ERROR, 'You can only edit your own plugins.', $Template);
 	$Template->SetRefresh('showplugin.php?id=' . $_GET['id']);
-	return;	
+	return;
 }
 
 if (isset($_POST['DeleteConfirmed' . $_GET['id']]))
 {
-	GitHubAPI::DeleteRepositoryUpdateHook($_GET['id'], $Query['UpdateHookID']);	
+	GitHubAPI::DeleteRepositoryUpdateHook($_GET['id'], $Query['UpdateHookID']);
 	$SQLLink->query('DELETE FROM PluginData WHERE RepositoryID = %i', $_GET['id']);
```

Comment:
This has nothing to do with this PR, but this looks like a possible SQL injection here. I can't get really wrap my head around the way the MeekroDB class works. I got no further than [this](https://github.com/cuberite/plugin-repository/blob/bb0832c4a3fa4469811278caf9d7e90986787a93/helpers/meekrodb.php#L589)